packages/scene-system/sdk/Fetch.ts

```diff
@@ -8,9 +8,13 @@ export interface FetchOptions {
   log(...a: any[]): void
 }
 
+type Opts = {
+  timeout?: number
+}
+
 export function createFetch({ canUseFetch, previewMode, log, originalFetch }: FetchOptions) {
   const fifoFetch = new PQueue({ concurrency: 1 })
-  return (resource: RequestInfo, init?: RequestInit | undefined): Promise<Response> => {
+  return (resource: RequestInfo, init?: RequestInit | undefined, opts?: Opts): Promise<Response> => {
```

Comment:
@gonpombo8 before releasing a stable version, may I suggest to add timeout to the second argument instead adding an extra arg?